testing/E2E_Test_Update.md

```diff
@@ -0,0 +1,163 @@
+proposed that best practices such as page objects should be used to avoid 
+code repetition and split the presentation logic from the test logic, and 
+that while the Appium Java-client was the preferred option to create the tests, 
+the proposal called for using Kotlin to create a domain specific language (DSL) 
+to write the tests, as it would provide easier readability and understanding 
+for new developers.
+ 
+### Test Infrastructure Proposal Summarry
+The test proposal looked at possible ways of running tests in a continuous 
+integration manner.   Tools such as Circle CI, Travis and Jenkins CI were 
+evaluated to determine their viability in running the appium UI tests.  It was 
+determined, due to problems running on Circle CI, Travis and docker images on 
+Jenkins CI, a macOS slave should be attached to Jenkins CI.  On this device 
+tools such as Appium, Android Emulators and iOS simulators can be installed and
+ created to provide the necessary environment for running the the Continuous 
+Integration end to end UI tests.  Devices could also be attached to the new 
+slave in order to to provide the capability of running tests on real devices.
+
+### Updates on Test Infrastructure
+Regarding the test infrastructure, a new macOS slave has been attached to the 
+Jenkins CI infrastructure.  The require software has been installed on the 
+device and emulators have been created.  As of yet no real devices have been 
+attached to the slave to provide the the ability to run the tests on real 
+devices. The only thing that needs to be checked is the compatability of the
+Chromedriver and the emulators installed on the macOS slave.
+
+## Making a Solid Test Suite
+
+Where possible, the use of preferred selectors such as AccessibilityId, Id, name etc., 
+placed inside the applications on relevant UI elements would be preferable, as it would 
+eliminate the risks of minor changes in the development of the application, such as moving 
+the order or position of elements on a page or changing text values, requiring an update 
+in the UI tests. 
+
+The use of common design in application development coupled with the development of the 
+suite of tests in a single repository that contains smart abstractions can be major factors 
+in the reduction of the amount of code that will be written and in turn that needs to be 
+maintained.  Common design of application means that only a single test needs to be written 
+to cover all platforms that need to be covered if abstractions such as page objects and 
+data objects are used to remove selectors and expected data from the the tests themselves.  
+These abstractions can then be passed into the tests for multiple platforms.  Indeed, the 
+Java-client has support for cross-platform page objects through the use of special 
+annotations.
+
+## Updated Proposals
+
+A decision should first be made if prefferred selectors are to be implmented in the mobile
```

Comment:
@twnolan there should be 'preferred' instead of 'prefferred' and 'implemented' instead of 'implmented'. Additionally, the next line contains a sentence started with the double space